groups/committee-code-of-conduct/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+    committee-code-of-conduct
```

Comment:
No approvers until we can ensure subdirs of groups can't create arbitrary groups

Reply:
Fixed.